pvlib/shading.py

```diff
@@ -232,3 +232,66 @@ def sky_diffuse_passias(masking_angle):
        Available at https://www.nrel.gov/docs/fy18osti/67399.pdf
     """
     return 1 - cosd(masking_angle/2)**2
+
+
+def projected_solar_zenith_angle(axis_tilt, axis_azimuth,
+                                 solar_zenith, solar_azimuth):
+    r"""
+    Calculate projected solar zenith angle in degrees.
```

Comment:
Could we have description in terms of a vector and a surface, which I'm guessing the projection is all about?

Reply:
Sorry, I think it's not clear.  I think you have to say at least you're projecting onto a vertical plane. (Or maybe that's only for the non-tracking case?)

Reply:
Did you check the last change to the definition proposed by Cliff? IIRC, there wasn't much time between the time I made the change and your comment ([docs](https://pvlib-python--1904.org.readthedocs.build/en/1904/reference/generated/pvlib.shading.projected_solar_zenith_angle.html)).

Anyway, I'm pretty sure (although my conception may be wrong), the projection plane isn't vertical for all cases. Figure 5 from [1] (in the docs too) may be of help.

If you still think the definition can be improved, can you state some concrete changes? I've got some difficulties seeing where to improve it.

Reply:
After reading a bit more I realize that it's not always projecting onto a vertical plane.

Reply:
Glad to hear that! I'll close this conversation then 😉